peer/store-utils.go

```diff
@@ -70,10 +70,18 @@ func GetPeers() ([]Peer, error) {
 			}).Error("Failed to unmarshal peer")
 			continue
 		}
+		if p.ID.String() == context.MyUUID.String() {
+			// Skip self's information
+			continue
+		}
 		peers[i] = p
+		i = i + 1
+	}
+	if i == 0 {
```

Comment:
i don't mean to say using i in for loop. i mean to say before for loop you are making empty slice of peers. so you can simply "return peers, nil". if there is peer in the cluster then peers slice will not have anything so it can be done by simply using "return peers, nil"  instead of using extra i variable.